bucketdownload/README.md

```diff
@@ -1,3 +1,86 @@
 # bucketdownload
 
-Work in progress.
+Downloads a file/blob from a configured [gocloud.dev/blob] bucket. Allows to
+set the directory, filename and permissions for the downloaded file. It will
```

Comment:
"Allows setting the directory, filename and permissions of the downloaded file".